packages/medusa/src/services/line-item.ts

```diff
@@ -363,6 +364,60 @@ class LineItemService extends BaseService {
 
     return itemTaxLineRepo.create(args)
   }
+
+  async clone(
+    ids: string | string[],
+    context: {
+      data?: { orderEditId?: string; order_id?: string | null }
+      options?: { setOriginalLineItemId?: boolean }
+    } = {}
+  ): Promise<LineItem[]> {
```

Comment:
suggestion: split context into two separate params for consistency

```suggestion
  async clone(
    ids: string | string[],
    data: { order_edit_id?: string; order_id?: string | null } = {},
    options: { setOriginalLineItemId?: boolean } = {}
  ): Promise<LineItem[]> {
```